packages/components/src/components/domainproperties/list/ListDesignerPanels.tsx

```diff
@@ -42,8 +42,17 @@ export class ListDesignerPanels extends React.PureComponent<Props, State> {
     constructor(props) {
         super(props);
 
+        let model = props.initModel;
+        if (props.initModel.name) {
```

Comment:
Why the check for a name here? Was this an attempt to check if the initModel is from an existing list vs a new list creation case? If so, we should use the isNew() function on the ListModel class.